src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -792,20 +820,31 @@ void BLEManagerImpl::HandleTXCharCCCDWrite(rsi_ble_event_write_t * evt)
 
     if (isIndicationEnabled)
     {
-        // Post an event to the CHIP queue to process either a CHIPoBLE Subscribe or Unsubscribe based on
-        // whether the client is enabling or disabling indications.
+        // If indications are not already enabled for the connection...
+        if (!bleConnState->subscribed)
         {
-            event.Type                    = DeviceEventType::kCHIPoBLESubscribe;
-            event.CHIPoBLESubscribe.ConId = 1; // TODO:: To be replaced by device mac address
-            err                           = PlatformMgr().PostEvent(&event);
+            bleConnState->subscribed = 1;
+            // Post an event to the CHIP queue to process either a CHIPoBLE Subscribe or Unsubscribe based on
+            // whether the client is enabling or disabling indications.
+            {
```

Comment:
bracket pair here and the indentation are not needed.

Reply:
removed bracket